docs/pages/concepts/how-liveblocks-works.mdx

```diff
@@ -48,9 +48,15 @@ and each room can have specific [permissions](/docs/rooms/permissions) and
 ## Integrations
 
 Integrations for specific libraries and frameworks to add Liveblocks-powered
-collaborative experiences to your product. Integrations are designed to serve
-various collaboration use cases such as text editors, comments, creative tools,
-forms, and more.
+collaborative experiences to your product:
+[JavaScript](/docs/api-reference/liveblocks-client),
+[React](/docs/api-reference/liveblocks-react),
+[Redux](/docs/api-reference/liveblocks-redux),
+[Zustand](/docs/api-reference/liveblocks-zustand),
+[Yjs](/docs/api-reference/liveblocks-yjs), and
+[Node.js](/docs/api-reference/liveblocks-node). Integrations are designed to
```

Comment:
I don't think we should put "Node.js" here. It's not an integration to build collaborative apps, it's just our package to authorize connections and handle permissions.